api/catalog/api/utils/help_text.py

```diff
@@ -0,0 +1,16 @@
+from typing import Iterable
+
+
+def make_comma_separated_help_text(items: Iterable, name: str) -> str:
+    """
+    Generate help text that wraps each category in backticks.
```

Comment:
We should document the params and the returned value.
```suggestion
    Generate help text that wraps each category in backticks.
    :param items: ...
    :param name: ...
    :return: ...
```

Reply:
Ah, and this also no longer describes `category` operations! I'll update this.